src/js/controllers/stateController.js

```diff
@@ -69,6 +69,12 @@ exports.getState = async (req, reply) => {
 exports.updateState = async (req, reply) => {
   try {
     const { id } = req.params;
+    const state = await State.findOne({
```

Comment:
You can avoid finding the updatable object by using the result of  await State.update()